### DISCUSSION TITLE
Toggle for my posts?

### DISCUSSION THREAD

**Discussion by Croworbit** [Tyrrrz/DiscordChatExporter#1128] [Q&A] [answered]
is there a toggle to export just my messages in a server?
I am joined on servers that have thousands of users and they tend to have been around for a very long while
I can't possibly have the storage to download every message ever, can there be/is there a way to just get my posts and no others?
and if possible, to delete them all after exporting?

**Comment by Tyrrrz** [comment]
Use message filters

**Comment by Croworbit** [comment]
I'll be honest, I have no idea if thats a setting in the program or how to acomplish that
I'm not trying to waste your time, I just might be a bit clueless

**Reply by 96-LB** [reply, marked as answer]
Check out the wiki - there is tons of guidance on how to use the program. [Here](https://github.com/Tyrrrz/DiscordChatExporter/blob/master/.docs/Message-filters.md) is the relevant page on message filters. You can use them with the `--filter` command in CLI or by writing a filter in the message filter box on GUI (you may have to expand the menu that pops up when you export a channel in order to see all the options).

**Reply by Croworbit** [reply]
thanks. I'm not the best at figuring this stuff out
